### PR TITLE
Add 64bit Min Sketch

### PR DESCRIPTION
**Pull request by jponf**
Added a new CMS with cell size uint64_t. This should address https://github.com/RaRe-Technologies/bounter/issues/51. 

To control the cell size I've added a new parameter so the user has a choice to use 32 or 64 bit. By default is set to 32 bit so old code should work exactly the same.

I'm open to address any issue or concern that you may have.

### CODE REVIEW

**Comment by piskvorky** [comment]
This looks good, thanks!

**Comment by jponf** [comment]
@piskvorky sorry my bad I didn't check that the CI runs on python 3.4 and 3.5, I've added f-strings that's probably why these are failing. As for 3.7 I have no clue because 3.6 seems ok I'll check it now.

**Comment by jponf** [comment]
@piskvorky I've fixed the string's formatting issue and it works on all versions except 3.7, looking at the logs it seems the error is comming from setuptools rather than the bounter package as it fails at the `setuptools` import line in `setup.py`

Maybe trying a newer version of 3.7 such as 3.7.9 instead of 3.7.1 fixes the CI pipeline.

```text
/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/setuptools/_importlib.py:23: UserWarning: `importlib-metadata` version is incompatible with `setuptools`.
This problem is likely to be solved by installing an updated version of `importlib-metadata`.
  warnings.warn(msg)  # Ensure a descriptive message is shown.
Traceback (most recent call last):
  File "setup.py", line 16, in <module>
    from setuptools import setup, find_packages, Extension
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/setuptools/__init__.py", line 18, in <module>
    from setuptools.dist import Distribution
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/setuptools/dist.py", line 34, in <module>
    from ._importlib import metadata
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/setuptools/_importlib.py", line 39, in <module>
    disable_importlib_metadata_finder(metadata)
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/setuptools/_importlib.py", line 12, in disable_importlib_metadata_finder
    import importlib_metadata
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/importlib_metadata/__init__.py", line 471, in <module>
    __version__ = version(__name__)
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/importlib_metadata/__init__.py", line 438, in version
    return distribution(package).version
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/importlib_metadata/__init__.py", line 411, in distribution
    return Distribution.from_name(package)
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/importlib_metadata/__init__.py", line 179, in from_name
    dists = resolver(name)
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/setuptools/_vendor/importlib_metadata/__init__.py", line 886, in find_distributions
    found = self._search_paths(context.name, context.path)
AttributeError: 'str' object has no attribute 'name'
```

**Comment by jponf** [comment]
I've tried it on my computer using python 3.7.1 installed via `pyenv` and it worked without further changes.

**Comment by piskvorky** [comment]
@mpenkov could you please check? This looks like a useful change, let's get it merged in and released. Many thanks.

**Review by piskvorky — APPROVED** [review]

**Comment by jponf** [comment]
Hi @mpenkov thanks for fixing the issue with Python 3.7.

Sadly now it failed in 3.6, but it is weird, as before all tests passed and all now all other versions are passing them too. My guess is that there is a rounding error that may or may not happen as the offending test is 

```python
self.assertAlmostEqual(self.cms['big number'], big_number, delta=self.delta * big_number)
```

Can you rerun https://app.travis-ci.com/github/RaRe-Technologies/bounter/jobs/587411973? If the error is due to floating point rounding maybe it was just bad luck that we had that rounding error.

**Comment by mpenkov** [comment]
Yeah, seems like a flickering test. Everything passes now. I'll leave you some minor comments, then we should be good to merge.

**Review by mpenkov — CHANGES_REQUESTED** [review]

File: bounter/count_min_sketch.py [hunk 1]
```diff
@@ -47,7 +53,14 @@ class CountMinSketch(object):
         counting as the collision bias will already be minimal.
     """
 
-    def __init__(self, size_mb=64, width=None, depth=None, log_counting=None):
+    def __init__(
+        self,
+        size_mb=64,
+        cell_size=CellSize.BITS_32,
```
[the remark this answers is not in the document]
Comment (jponf) [reply bounter/count_min_sketch.py]: Done

**Comment by mpenkov** [comment]
Merged! Thank you for your work @jponf

**Pull request merged**